selfdrive/controls/tests/test_leads.py

```diff
@@ -29,7 +29,7 @@ def single_iter_pkg():
     states = [m for m in out if m.which() == "radarState"]
     failures = [not state.valid and len(state.radarState.radarErrors) for state in states]
 
-    self.assertTrue(len(states) == 0 or all(failures))
+    self.assertTrue(all(failures))
```

Comment:
if states was previously empty, this will still allow that (all on an empty list is true)